storagenode/storagenodedb/diskspaceusages.go

```diff
@@ -0,0 +1,143 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedb
+
+import (
+	"context"
+	"database/sql"
+	"time"
+
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storagenode/console"
+)
+
+// diskSpaceUsage works disk space usage db
+// which caches data from satellite rollups
+type diskSpaceUsage struct {
+	*InfoDB
+}
+
+// Store stores disk space usage stamps to db
+func (db *diskSpaceUsage) Store(ctx context.Context, stamps []console.DiskSpaceUsage) (err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	if len(stamps) == 0 {
+		return nil
+	}
+
+	stmt := `INSERT OR REPLACE INTO disk_storage_usages(satellite_id, at_rest_total, timestamp) 
+			VALUES(?,?,?)`
+
+	cb := func(tx *sql.Tx) error {
+		txStmt, err := tx.PrepareContext(ctx, stmt)
```

Comment:
Why bother with the statements? AFAIK, it doesn't have any significant performance benefits and usually ends up making the code more complicated.